examples/assets/bash_workflow_template.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "QE SCF Calculation",
+  "subworkflows": [
+    {
+      "name": "Total energy",
+      "application": {
+        "_id": "ZcxX9DDY7GWcQKkEj",
+        "name": "shell",
+        "version": "4.2.46",
+        "build": "Default",
+        "isDefault": true,
+        "summary": "Shell Script",
+        "shortName": "sh",
+        "updatedBy": "0"
+      },
+      "units": [
+        {
+          "type": "execution",
+          "application": {
+            "_id": "ZcxX9DDY7GWcQKkEj",
+            "name": "shell",
+            "version": "4.2.46",
+            "build": "Default",
+            "isDefault": true,
+            "summary": "Shell Script",
+            "shortName": "sh",
+            "updatedBy": "0"
+          },
+          "results": [],
+          "monitors": [
+            {
+              "name": "standard_output"
+            }
+          ],
+          "head": true,
+          "input": [
+            {
+              "content": "",
+              "name": "job_script.sh",
+              "applicationName": "shell",
+              "executableName": "sh"
+            }
+          ],
+          "executable": {
+            "isDefault": true,
+            "monitors": [
+              "standard_output"
+            ],
+            "results": [
+              "atomic_forces",
+              "band_gaps",
+              "band_structure",
+              "density_of_states",
+              "fermi_energy",
+              "phonon_dispersions",
+              "phonon_dos",
+              "pressure",
+              "stress_tensor",
+              "total_energy",
+              "total_energy_contributions",
+              "total_force",
+              "zero_point_energy",
+              "final_structure",
+              "magnetic_moments",
+              "reaction_energy_barrier",
+              "reaction_energy_profile",
+              "potential_profile",
+              "charge_density_profile"
+            ],
+            "name": "sh"
+          },
+          "flavor": {
+            "isDefault": true,
+            "input": [
+              {
+                "name": "hello_world.sh"
```

Comment:
Is this `hello_world.sh` file a placeholder?

Reply:
Yea, file name is not used by the cluster, just the code from this file is executed. However, it is shown in the web UI, changed to `job.sh`.